docs/assets/custom.css

```diff
@@ -34,3 +34,8 @@
   vertical-align: middle;
   margin-left: 0.3em;
 }
+
+/*  From https://github.com/squidfunk/mkdocs-material/discussions/6404#discussioncomment-7695243 */
+.md-main__inner.md-grid {
+  max-width: initial;
```

Comment:
would there be a way to only apply this to the cookbook pages? I am not sure if I really like this setting for other pages (but I am ok with going with this for now)